### PR TITLE
implement support for multi package repo WIP

### PR DESCRIPTION
**Pull request by remonh87li**

### CODE REVIEW

File: rosetta/lib/src/stone.dart [hunk 1]
```diff
@@ -6,10 +6,15 @@ class Stone {
...
   /// Must not be [null].
   final String path;
+  /// In case of a mono repo approach you can add the package
+  /// where the translation file is located.
+  /// Can be [null]
+  final String package;
 
   /// Create an annotation that will generate the Helper and
-  /// Delegate classes for the translations located at [path].
+  /// Delegate classes for the translations located at [path] and/or [package].
```
Comment (werediver) [inline rosetta/lib/src/stone.dart]: What happens if both parameters are specified? Should it be explained in the doc-comment?

File: rosetta_generator/lib/src/rosetta_helper.dart [hunk 2]
```diff
@@ -37,9 +37,10 @@ Class generateHelper(String className, String path,
   );
 }
 
-Method generateLoader(String path) {
+Method generateLoader(String package, String path) {
   var assetLoader = refer("rootBundle").property("loadString");
   var decodeJson = refer("json").property("decode");
+  var packageString = package.isNotEmpty ? "packages/$package" : "";
```
Comment (werediver) [inline rosetta_generator/lib/src/rosetta_helper.dart]: Should it be `packagePath`? The `-String` suffix doesn't bring a lot of meaning.

File: rosetta/lib/src/stone.dart (hunk 1 above)
Comment (remonh87li) [reply rosetta/lib/src/stone.dart]: fixed

File: rosetta_generator/lib/src/rosetta_helper.dart (hunk 2 above)
Comment (remonh87li) [reply rosetta_generator/lib/src/rosetta_helper.dart]: indeed that is a better name

**Pull request merged**